assets/app/view/game_card.rb

```diff
@@ -191,35 +191,62 @@ def render_body
 
       p_elm = players.map.with_index do |player, index|
         short_name = player['name'].truncate
+        player_props = { attrs: { title: player['name'].to_s } }
+        button_style = {
+          padding: '0.1rem 0.3rem',
+          margin: '0 0.3rem 0.1rem 0.3rem',
+        }
+
+        p_content = []
+
+        if index.positive? && (owner? && new? && !@gdata.dig('settings', 'player_order').nil?)
```

Comment:
you don't need the nil

Reply:
I don't even need the dig. I just thought it would make things more clear